tools/block-generator/README.md

```diff
@@ -34,6 +35,28 @@ asset_optin_fraction: 0.1
 asset_close_fraction: 0.05
 asset_xfer_fraction: 0.849
 asset_delete_fraction: 0
+
+# app choice config
+app_swap_fraction: 0.5
+app_boxes_fraction: 0.5
+
+# app_swap config
+app_swap_create_fraction: 0.001
+app_swap_update_fraction: 0.001
+app_swap_delete_fraction: 0
+app_swap_optin_fraction: 0.1
+app_swap_call_fraction: 0.98
+app_swap_close_fraction: 0.005
+app_swap_clear_fraction: 0.003
```

Comment:
could you please elaborate what you have mind for an app swap scenario? are these apps using the swap opcode?

Reply:
This is out of context. #5450  sheds a bit of light. In particular, I'm creating two apps with teal code:
1. for [populating boxes](https://github.com/algorand/go-algorand/blob/24530a17d4a03ba4c11efce6596848db6bdce50c/tools/block-generator/generator/teal/poap_boxes.teal)
2. for [swapping assets using inner transactions](https://github.com/algorand/go-algorand/blob/24530a17d4a03ba4c11efce6596848db6bdce50c/tools/block-generator/generator/teal/swap_amm.teal)

That PR doesn't actually make boxes / swap assets yet, but it does create the apps and set some local state.